tobac/plotting.py

```diff
@@ -561,54 +562,9 @@ def plot_mask_cell_track_follow(
     dpi=300,
     **kwargs
 ):
-    """Make plots for all cells centered around cell.
-
-    With one background field as filling and one background field as
-    contours.
-
-    Parameters
-    ----------
-    cell : int
-        Integer id of cell to create masked cube for.
-
-    track
-
-    cog
-
-    features : pandas.DataFrame
-        Output from trackpy/maketrack.
-
-    mask_total : iris.cube.Cube
-
-    field_contour
-
-    field_filled
-
-    width : int, optional
-        Default is 10000.
-
-    name : str, optional
-        Default is 'test'.
-
-    plotdir : str, optional
-        Path where the plot will be saved. Default is './'.
-
-    file_format : {['png'], ['pdf']}, optional
-        Default is ['png'].
-
-    dpi : int, optional
-        Plot resolution. Default is 300.
-
-    **kwargs
-
-    Returns
-    -------
-    none
-
-    Notes
-    -----
-    unsure about features, mask_total
-    needs more descriptions
+    """Make plots for all cells centred around cell and with one background field as filling and one background field as contrours
+    Input:
+    Output:
```

Comment:
For this function, you reversed the docstring changes. Just checking if this was intentionally?